documentation/docs/student/calendar.mdx

```diff
@@ -0,0 +1,86 @@
+---
+id: calendar
+title: Calendar
+sidebar_label: Calendar
+---
```

Comment:
Can we make sure that the calendar feature is explained after the `General Information`?